pybossa/api/__init__.py

```diff
@@ -461,3 +463,63 @@ def task_gold(project_id=None):
     task_repo.update(task)
 
     return Response(json.dumps({'success': True}), 200, mimetype="application/json")
+
+
+@jsonpify
+@login_required
+@csrf.exempt
+@blueprint.route('/task/<task_id>/services', methods=['POST'])
+@ratelimit(limit=ratelimits.get('LIMIT'), per=ratelimits.get('PER'))
+def get_service_request(task_id):
+    """Proxy service call"""
+    proxy_service_config = current_app.config.get('PROXY_SERVICE_CONFIG', None)
+    task = task_repo.get_task(task_id)
+    payload = request.json
+
+    if not task or not proxy_service_config:
+        return abort(400)
+
+    scheduler, timeout = get_project_scheduler_and_timeout(
+        task.project_id)
```

Comment:
proj timeout can also be obtained from project info, and if its not set, use default
> project['info'].get('timeout', ContributionsGuard.STAMP_TTL)

Reply:
agree